tests/boot/boot_from_pxe.pm

```diff
@@ -51,7 +51,7 @@ sub run() {
 
         my $arch = get_var('ARCH');
         my $path = "/mnt/openqa/repo/${image_name}/boot/${arch}/loader";
-        my $repo = get_required_var('OPENQA_URL') . "/assets/repo/${image_name}";
+        my $repo = 'http://openqa.suse.de' . "/assets/repo/${image_name}";
```

Comment:
because that's where the assets are - you can't run the test cases anywhere else otherwise. But I will revert once it works :)